plugins/tiddlywiki/katex/wrapper.js

```diff
@@ -16,6 +16,23 @@ var katex = require("$:/plugins/tiddlywiki/katex/katex.min.js"),
     chemParse = require("$:/plugins/tiddlywiki/katex/mhchem.min.js"),
 	Widget = require("$:/core/modules/widgets/widget.js").widget;
 
+katex.macros = {};
+katex.updateMacros = function() {
+	var tiddlers = $tw.wiki.getTiddlersWithTag("$:/tags/katex/macro"),
+		regex = /#\d/g, // Remove the arguments like #1#2
+		tid, macro, cmd;
+	for (var i=0; i < tiddlers.length; i++) {
+		tid = $tw.wiki.getTiddler(tiddlers[i]);
+		try {
+			macro = tid.fields["macro"];
+			macro = macro.replace(regex, "");
```

Comment:
Why do we remove the `#1`, `#2` from the "macro" string?

Reply:
The KaTeX macro variable does not accept inputs like `\dd#1#2`, however I think it is more natural to allow this in the definition, since usually one writes `\def\dd#1#2{\frac...}`. So now both `\dd` and `\dd#1#2` can be put in the macro editor, and KaTeX will only get `\dd` in either case.